index.htm

```diff
@@ -649,8 +649,8 @@ <h3>Isomorphic keyboard settings</h3>
                 <input id="input_modify_mode" name="" type="text" class="form-control" value=""
                     placeholder="2 2 1 2 2 2 1"></input>
                 <div class="form-row">
-                    <button class="btn btn-default" id="input_mode_step_left" type="button">&laquo</button>
-                    <button class="btn btn-default" id="input_mode_step_right" type="button">&raquo</button>
+                    <button class="btn btn-default btn-thinner" id="input_mode_step_left" type="button">&laquo</button>
```

Comment:
I tested this out, replacing "btn-thinner" with "btn-smaller", and it didn't change it at all. I tried some other combinations with and without "btn-default" but none of them did what I was going for. I think the "btn-sm" scales it down proportionally, but I'm looking to just reduce the height.

Reply:
On a closer look, "btn-sm" changes the padding to be 5px at the top/bottom, and 10px on left/right. However, the text has a margin that is blocking that from being effective. I'll see if I can fix that somehow so I can use "btn-sm".

Reply:
![image](https://user-images.githubusercontent.com/13040970/77371165-102a6a00-6d39-11ea-9335-63438f86df89.png)
I think there's a jQuery + Bootstrap conflict, since the margin area lights up when I hover over this jQuery style (with a dark blue background)
![image](https://user-images.githubusercontent.com/13040970/77371283-597ab980-6d39-11ea-9427-d74b99472a6d.png)

Reply:
Thank you for looking into it, I think we can go with the original btn-thinner solution then. The conflict comes from jQueryUI and it might need a separate look to resolve conflicting styles like this.